doc/architecture/decisions/0036-use-csv-safe-to-process-csv-files.md

```diff
@@ -0,0 +1,21 @@
+# 36. Use csv-safe to process CSV files
+
+Date: 2023-03-16
+
+## Status
+
+Accepted
+
+## Context
+
+RODA generates CSV files containing user input. If this user input contains malicious code, it could compromise the machines of the users opening CSV files in applications such as MS Excel.
+
+## Decision
+
+We will use the gem `csv-safe` as a replacement for the `csv` gem when generating CSV files.
+
+## Consequences
+
```

Comment:
Maybe this file should say that you had to monkey patch the library to handle currencies?